src/App.js

```diff
@@ -1,16 +1,30 @@
 import React, { Component } from "react";
-
+import Heading from './components/Heading'
 import Bookings from "./Bookings";
```

Comment:
Why are some components in `src/components/` and others in `src/`?  Your reader (your coworkers, or future you) will waste time thinking about this, and will waste time trying to remember which components are where.
Be consistent.